vscode/src/extension.ts

```diff
@@ -156,10 +158,18 @@ function registerDocumentUpdateHandlers(languageService: ILanguageService) {
  * Loads the Q# compiler including the WASM module
  */
 async function loadLanguageService(baseUri: vscode.Uri) {
+  const start = performance.now();
   const wasmUri = vscode.Uri.joinPath(baseUri, "./wasm/qsc_wasm_bg.wasm");
   const wasmBytes = await vscode.workspace.fs.readFile(wasmUri);
   await loadWasmModule(wasmBytes);
-  return await getLanguageService();
+  const end = performance.now();
```

Comment:
Shouldn't this be after you await getting the language service (line below) if you're timing how long the language service takes to load?